### PR TITLE
Add racket support

### PR DESCRIPTION
**Pull request by 6cdh**
Neovim don't support racket filetype currently.
It's necessary to add filetype detect manually or use [vim-racket](https://github.com/wlangstroth/vim-racket) plugin.

### CODE REVIEW

**Review by kyazdani42 — APPROVED** [review]
let's wait for another review before merging :)

**Comment by theHamsta** [comment]
Looks good! Thank you!

**Review by theHamsta — APPROVED** [review]

**Pull request merged**
